google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -73,7 +93,10 @@ public void run() {
           loggingHandler,
           "Created temporary staging directory: " + stagingDirectory.getAbsolutePath());
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      logger.error(e);
```

Comment:
Since your are logging the error here, for consistency should we also log it in the other error handling blocks?